powershell/VstsTaskSdk/InputFunctions.ps1

```diff
@@ -1,3 +1,5 @@
+$script:secureInputs = @{ }
```

Comment:
This is a new private module-scoped variable that stores the secured input values.